msteams-platform/whats-new.md

```diff
@@ -24,16 +24,9 @@ Discover Microsoft Teams platform features that are generally available (GA). Yo
 
 Teams platform features that are available to all app developers.
 
-**2023 February**
+**2023 March**
 
-* ***February 28, 2023***: [Facilitate adoption of your app and create awareness](promote-app-adoption.md)
```

Comment:
is this change intentional?

Reply:
Yes @tacri, 

We've updated the What's new file:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/75663981/226030527-68f59922-44ac-45ab-aa27-1644ee2a3162.png">